src/types.ts

```diff
@@ -499,4 +503,19 @@ export default {
    */
   // eslint-disable-next-line sort-keys
   any,
+
+  /**
+   * This allows any value to be assigned, but is typed as unknown to force assertions
+   * before relying on the data. Like with `any`, we recommend avoiding this type.
+   * It only exists as an escape hatch for unknown data.
+   *
+   * @example
+   * import { schema, types } from 'papr';
+   *
+   * schema({
+   *   unknownData: types.unknown(),
+   * });
+   */
+  // eslint-disable-next-line sort-keys
```

Comment:
Let's move this entry later in the object to keep the keys sorted.

Reply:
Will do. I'm away from my desk but this _might_ require moving the `any` property up as well - I actually think without that property at the bottom (just above `unknown`) this will be in the correct place.

Reply:
@avaly as expected, `unknown` is actually in the correct position. The disable tag wasn't necessary and has been removed. The `any` above it is out of order but it feels like out of scope to address in this PR.